src/plugins/kibana_utils/public/state_management/url/format.ts

```diff
@@ -22,6 +22,23 @@ import { stringify, ParsedQuery } from 'query-string';
 import { parseUrl, parseUrlHash } from './parse';
 import { url as urlUtils } from '../../../common';
 
+export function replaceUrlQuery(
```

Comment:
I don't think you should change the content of `kibana_utils`

Reply:
That change is now merged in here https://github.com/elastic/kibana/pull/74955.